src/seedu/addressbook/parser/Parser.java

```diff
@@ -73,33 +63,36 @@ public Command parseCommand(String userInput) {
 
         switch (commandWord) {
 
-        case AddCommand.COMMAND_WORD:
-            return prepareAdd(arguments);
+            case AddCommand.COMMAND_WORD:
```

Comment:
This indentation for switch statements violates the [coding standard](https://google.github.io/styleguide/javaguide.html#s4.8.4-switch).